wifi-qr

```diff
@@ -90,10 +92,10 @@ call_wifi_gui() {
 }
 
 call_qr_gui() {
-    call_wifi_pass
-    QR_PNG="/tmp/wifi-qr-$(echo $WIFIS | tr -dc '[:alnum:]')"
-    qrencode -l h -s 14 -o $QR_PNG "WIFI:S:$WIFIS;P:$KEEY;$PSSK$H;"
-    xdg-open $QR_PNG
+  call_wifi_pass
+  QR_PNG="/tmp/wifi-qr-$(echo $WIFIS | tr -dc '[:alnum:]').png"
+  qrencode -l h -s 14 -o $QR_PNG "WIFI:S:$WIFIS;P:$KEEY;$PSSK$H;"
+  xdg-open $QR_PNG
```

Comment:
Most image viewer could handle without file format. But for shotwell, it will return error without file format .png

Reply:
So other addition function like more interest ?
save at $Pictures/wifi-qr/ 
i am currently use for tmp and remove png to after use 
at last script

Reply:
So it can do something like internal saved picture scan-connect with additional function? It would be good use case for connect later when wifi available.

Reply:
Actually it's share to other.